scripts/sct_download_data.py

```diff
@@ -214,4 +215,6 @@ def download_data(urls, verbose):
 
 if __name__ == "__main__":
     sct.init_sct()
-    main()
+    res = main()
+    raise SystemExit(res)
```

Comment:
This will change the exception `KeyboardInterrupt` for a `SystemExit`, it might confuse error reporting, exception handling, etc. Please see the #1658. Let's say someone uses `sct_download_data` later (as an API), it will silence the `KeyboardInterrupt` exception in favor of a C-style error reporting of returning a code.
If there is any other way of doing that it would be nice.

Reply:
The ``main()`` function was not exactly behaving as an API-callable function (should have one named ``download_data()`` or similar) but rather like a ``main()`` function... 
As the user could ^C anytime, I just removed the try/catch and the user will get a standard traceback message, as in every other SCT script.